neo/Ledger/MemoryPool.cs

```diff
@@ -448,6 +449,9 @@ internal void UpdatePoolForBlockPersisted(Block block, Snapshot snapshot)
                 int blocksTillRebroadcast = unverifiedSortedTxPool == _sortedHighPrioTransactions
                     ? BlocksTillRebroadcastHighPriorityPoolTx : BlocksTillRebroadcastLowPriorityPoolTx;
 
+                if (Count > RebroadcastMultiplierThreshold)
+                    blocksTillRebroadcast = blocksTillRebroadcast * Count / RebroadcastMultiplierThreshold;
```

Comment:
I will take another look to understand, capacity is quite big.

Reply:
Capacity is not fixed, Jeff?
I did not exactly got this.

Currently, Capacity is 50000, right?
Thus `RebroadcastMultiplierThreshold = 5000` kind of fixed, right?

Then, for example.
If Count = 1000
blocksTillRebroadcast = 2 for high priority
blocksTillRebroadcast = 6 for low priority

If Count = 10000
blocksTillRebroadcast = 20 for high priority
blocksTillRebroadcast = 60 for low priority

Sounds reasonable.

Reply:
It just makes a linear progression as the pool fills up to in our current case the max Capacity of 50000 makes RebroadcastMultiplierThreshold=5000

For a capacity of 5000 or less:
* 30 blocks for rebroadcasting low priority
* 10 blocks for rebroadcasting high priority

From then on for every additional 5000 in the pool the time linearly increases by the same amount, so...

at 10000 in the pool:
* 60 blocks for rebroadcasting low priority 
* 20 blocks for rebroadcasting high priority

at 15000 in the pool:
* 90 blocks for low
* 30 blocks for high
...
at 50000 (full pool)
* 300 blocks for rebroadcasting low
* 100 blocks for rebroadcasting high

Also note that it doesnt jump at 5000 levels, for example @ 6000
* 36 blocks for rebroadcasting low priority
* 12 blocks for rebroadcasting high

Reply:
Our comments were almost together...aehauehauea

Reply:
Yeah, you got it

Reply:
I did not get this exception of `For a capacity of 5000 or less:`

Reply:
Are you going to push it or it is already there?

Reply:
It’s already there

Reply:
Anyway, sounds good to me, Jeff.
Congratulations. Nice idea.

Reply:
`It’s already there`, I did not see...aehuaheuaea